src/gui/folder.cpp

```diff
@@ -416,9 +416,9 @@ void Folder::createGuiLog(const QString &filename, LogStatus status, int count,
             break;
         case LogStatusNew:
             if (count > 1) {
-                text = tr("%1 and %n other file(s) have been downloaded.", "", count - 1).arg(file);
+                text = tr("%1 and %n other file(s) are new.", "", count - 1).arg(file);
```

Comment:
I think `are new` is a bit strange.
maybe `have been added` or `have been created` 
What do you think?

Reply:
Yeah, updated